js/index.js

```diff
@@ -0,0 +1,106 @@
+// 1
+const formalgGreeting = 'Hello, ';
+const informalgGreeting = 'Hi, ';
+const exclamation = '!';
+const favouriteUser = 'Mark'
+
+const fullPhrase = function (inputName) {
+    if (inputName === favouriteUser){
+    return `${informalgGreeting}${inputName}${exclamation}`;
+    }
+    else {
+        return `${formalgGreeting}${inputName}${exclamation}`;
+    }
+};
+console.log(fullPhrase('Mark'));
+
+
+// 2 не очень понятно само задание, ведь для вычисления гипотенузы нужны две стороны, а нам даны три, поэтому для формулы я взяла одну сторону - 3 и вторую сторону - 4
+const firstSide = 3;
+const secondSide = 4;
+
+const thirdSide = function () {
+    return Math.sqrt(firstSide ** 2 + secondSide ** 2);
+}
+console.log (thirdSide());
+
+
+// 3
+const compareFunction = function (a, b) {
+    if (a > b) {
+        return b;
+    }
+    if (a < b) {
+        return a;
+    }
+};
+console.log (compareFunction(-1, 4));
+
+
+// 4
+const isEven = function (number) {
+    if (number % 2 === 0) {
+        return true;
+    }
+    else {
```

Comment:
1. fix issue with redundant else here and in all places below
2. in this case if is redundant. `return number % 2 === 0` will be enough, just because this comparison will return true or false anyway